src/cascadia/TerminalApp/Jumplist.cpp

```diff
@@ -204,7 +146,7 @@ winrt::fire_and_forget Jumplist::UpdateJumplist(const CascadiaSettings& settings
     {
         auto sh = winrt::create_instance<IShellLinkW>(CLSID_ShellLink, CLSCTX_ALL);
 
-        const auto module{ _getExePath() };
+        const auto module{ GetWtExePath() };

```

Comment:
FYI: `_getExePath` was an identical copy of `GetWtExePath`. I deduplicated the code, as I was touching `WtExeUtils.h` anyways already.

Reply:
thanks for fixing this